client/js/components/user/orgaRegisterForm/OrgaRegisterForm.vue

```diff
@@ -52,31 +52,37 @@
             <legend :class="prefixClass('font-size-medium is-label u-mb-0_25')">
               {{ Translator.trans('organisation.type') }}
             </legend>
-            <div :class="hasPermission('feature_identity_broker_login') ? prefixClass('space-stack-xs') : prefixClass('o-form__group')">
+            <p :class="prefixClass('u-mb')">
+              {{ Translator.trans('organisation.kind.explanation') }}
+            </p>
+            <div :class="hasPermission('feature_identity_broker_login') ? prefixClass('space-stack-xs') : prefixClass('o-form__group flex-col')">
               <dp-checkbox
                 id="orgatype_invitable_institution"
                 data-cy="orgatype_institution"
-                :class="prefixClass('o-form__group-item')"
+                :class="prefixClass('o-form__group-item u-mb-0_75 u-ml-0')"
                 :label="{
-                  text: Translator.trans('invitable_institution')
+                  text: Translator.trans('invitable_institution'),
+                  hint: Translator.trans('register.institution.hint')
                 }"
                 name="r_orgatype[]"
                 value-to-send="OPSORG" />
               <dp-checkbox
                 id="orgatype_municipality"
                 data-cy="orgatype_municipality"
-                :class="prefixClass('o-form__group-item')"
+                :class="prefixClass('o-form__group-item u-mb-0_75 u-ml-0')"
```

Comment:
Why do we need to set `u-ml-0` here? The `o-form__group-item` class is (somewhat beta-style) used for horizontally layed out items, it can be removed here i guess, alongside the margin.

Reply:
thx, done here https://github.com/demos-europe/demosplan-core/pull/2045/commits/7b10dcc10ab897b309f3438f1a7a055f3eb65660